src/odemis/dataio/tiff.py

```diff
@@ -1900,10 +1997,15 @@ def _saveAsMultiTiffLT(filename, ldata, thumbnail, compressed=True, multiple_fil
     # TODO: to keep the code simple, we should just first convert the DAs into
     # 2D or 3D DAs and put it in an dict original DA -> DAs
 
-    # Write the OME metadata + ImageJ metadata on the first image (only)
-    # If thumbnail is present, the tiff file is not compatible with ImageJ
+    imagej_compat = False # TODO: add a format to enable/disable ImageJ compatibility
+    if imagej_compat:
+        # Extract ImageJ compatible Image description based on the first group dimension
+        imagej_description = extract_imagej_metadata(sorted_groups[0][1])
+        # TODO: add imagej_description to the image description instead of ometxt
+
+    # Write the OME metadata in the first image
     if ometxt:
-        f.SetField(T.TIFFTAG_IMAGEDESCRIPTION, imagej_description.encode('ascii') + ometxt)
```

Comment:
We cannot drop the current feature that supports ImageJ.
Please keep this, under a new "imagej" parameter. Then, you can add a new "file format", name: `ij_tiff.py`, extension: `.ij.tiff`.

Reply:
added